ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -256,8 +256,14 @@ export default class AdvancedTab extends PureComponent {
             <Button
               type="secondary"
               large
-              onClick={() => {
-                this.props.backupUserData();
+              onClick={async () => {
+                const { fileName, data } = await this.props.backupUserData();
+                exportAsFile(fileName, data);
+
+                this._trackMetaMetricsEvent({
```

Comment:
This method doesn't exist, and seems to be the background method name, you'll need to use the context to get access to the frontend tracking method.

Reply:
Oh shoot. Yes. Thanks.